controllers/velero.go

```diff
@@ -597,6 +597,13 @@ func (r *DPAReconciler) customizeVeleroContainer(dpa *oadpv1alpha1.DataProtectio
 			Name:  "VOLUME_SNAPSHOT_MOVER",
 			Value: "true",
 		})
+
+		if len(dpa.Spec.Features.DataMover.Timeout) > 0 {
+			veleroContainer.Env = append(veleroContainer.Env, corev1.EnvVar{
+				Name:  "DATAMOVER_TIMEOUT",
```

Comment:
Do we wanna for consistency call volume_snapshot_mover_timeout?

Reply:
We are not using this timeout inside Volume_Snapshot_Mover, we are using this timeout inside datamover csi plugin, hence the name.